### PR TITLE
me when I mass PR fast

### LINKED ISSUE #36: private way to leave feedback
a /suggestion command to privately leave feedback would be a good idea as some may be uncomfortable leaving open feedback in the discord forums

it would work like /suggest [user message] and the message would be sent into mod chat/a mod only area

### PR DESCRIPTION
**Pull request by FoolOfMadness**
closes #36 and #30 

so many things, timestamp command, suggestion command, and ephemeral messages for errors

Can u pwease merge my pwull wequest senpai?!! UwU

### CODE REVIEW

File: commands/lorelei/magic8ball.js [hunk 1]
```diff
@@ -70,14 +70,17 @@ const execute = async (interaction) => {
...
 
-    //pig-pen/bot-spam bully check
+    //pig-pen/bot-spam & oinker bully check
     const channel = interaction.channel.id;
     let availableResponses = [];
-    if (channel === '1247276527125794867' || '1283447730693345353') {
-      //use all responses in pig-pen & bot-spam
+    if (
+      interaction.member.roles.cache.has('1256247572582891542') &&
+      (channel === '1247276527125794867' || '1283447730693345353')
```
[the remark this answers is not in the document]
Comment (FoolOfMadness) [reply commands/lorelei/magic8ball.js:78]: yes

File: commands/lorelei/suggestion.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
+//suggestion command
+const { SlashCommandBuilder, EmbedBuilder } = require('discord.js');
+
+//name of slash command & description
+const data = new SlashCommandBuilder()
+  .setName('suggestion')
```
[the remark this answers is not in the document]
Comment (FoolOfMadness) [reply commands/lorelei/suggestion.js:6]: yes

**Comment by FoolOfMadness** [comment]
> Need to change at least one thing, also in the future it would be best to have one feature/bug fix per pull request :sparklelei:

show me how to do this :nyaSpin:

**Review by FrostiestBunny — APPROVED** [review]
Good stuff

**Pull request merged**